mne/source_space.py

```diff
@@ -125,18 +125,36 @@ def plot(self, head=False, brain=None, skull=None, subjects_dir=None,
         fig : instance of mlab Figure
             The figure.
         """
+        from .viz import plot_alignment
+        surfaces = list()
+        bem = None
         if brain is None:
             brain = 'white' if any(ss['type'] == 'surf'
                                    for ss in self) else False
+        if isinstance(brain, str):
+            surfaces.append(brain)
+        elif brain:
+            surfaces.append('brain')
         if skull is None:
             skull = False if self.kind == 'surface' else True
-        from .viz import plot_trans
+        if isinstance(skull, str):
+            surfaces.append(skull)
+        elif skull is True:
+            surfaces.append('outer_skull')
+        elif skull is not False:  # list
+            if isinstance(skull[0], dict):  # bem
+                surfaces.append('inner_skull', 'outer_skull', 'outer_skin')
```

Comment:
Should check to make sure these are there. Also it could be single-layer, so better only to add the ones that are present.